frontend/src/pages/team/Members/Invitations.vue

```diff
@@ -3,9 +3,9 @@
         <ff-loading v-if="loading" message="Loading Invitations..." />
         <form v-else class="space-y-6">
             <div class="text-right"></div>
-            <ff-data-table :columns="inviteColumns" :rows="invitations">
+            <ff-data-table data-el="invites-table" :columns="inviteColumns" :rows="invitations">
```

Comment:
Same as earlier, I'm not sure how this is RBAC?

Reply:
data-el, data-action etc are attributes for e2e tests Ben. 

So in effect, RBAC related (1 test around RBACs was included in this PR as a template for the others)

We can of course pull the e2e test out and add it to Bens task #867 if more suited?

see here: https://github.com/flowforge/handbook/blob/main/development/frontend/data-attributes.md

Reply:
As above, leave it in